Improbable/Stdlib/Improbable.Stdlib/EventProcessor.cs

```diff
@@ -0,0 +1,55 @@
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Threading;
+using System.Threading.Tasks;
+using Improbable.Worker.CInterop;
+
+namespace Improbable.Stdlib
+{
+    public class EventProcessor<T> : IOpProcessor where T : struct
```

Comment:
Is this the most specific generic constraint?

Reply:
Yeah, there's no inheritance or anything else to go against